source/database/mssql/scripts/dbscripts/PSP_PIMS_S53_00/Alter Up/012_DML_PIMS_ROLE_CLAIM.sql

```diff
@@ -229,8 +229,10 @@ VALUES
     (@prjfunc, @projectAdd,                    N'SEED', @appUserGuid, N'SEED', @appUserGuid, '', ''),
     (@prjfunc, @projectEdit,                    N'SEED', @appUserGuid, N'SEED', @appUserGuid, '', ''),
     (@prjfunc, @projectDelete,                    N'SEED', @appUserGuid, N'SEED', @appUserGuid, '', ''),
-        -- Project Functional
+    (@prjfunc, @propertyView,                    N'SEED', @appUserGuid, N'SEED', @appUserGuid, '', ''),
```

Comment:
this has been cleared by the BA's apparently this was the desired functionality.